src/applications/accredited-representative-portal/containers/POARequestsPage.jsx

```diff
@@ -1,17 +1,46 @@
 import React from 'react';
-
+import { useLoaderData, NavLink, Outlet } from 'react-router-dom';
 import DigitalSubmissionAlert from '../components/DigitalSubmissionAlert/DigitalSubmissionAlert';
-import POARequestsTableFetcher from '../components/POARequestsTableFetcher/POARequestsTableFetcher';
-import usePOARequests from '../hooks/usePOARequests';
 
 const POARequestsPage = () => {
+  const poaRequests = useLoaderData();
+
   return (
     <>
       <h1 data-testid="poa-requests-heading">Power of attorney requests</h1>
       <DigitalSubmissionAlert />
       <div className="poa-requests-page-table-container">
-        <h2 data-testid="poa-requests-table-heading">Requests</h2>
-        <POARequestsTableFetcher usePOARequests={usePOARequests} />
+        {poaRequests.length === 0 ? (
+          <p data-testid="poa-requests-table-fetcher-no-poa-requests">
+            No POA requests found
+          </p>
+        ) : (
+          <>
+            <div role="tablist" className="poa-request__tabs">
+              <NavLink
+                to="/poa-requests/pending"
```

Comment:
These tabs contribute to the query that we use to search for POA requests. This query will be reflected in the query string parameters like `/poa-requests?status=pending`.

Reply:
I reached out to kevin about this prior to submitting pr since params were not working for me at all. I left it like this so the tabs would actually work for my video example. We don't have an endpoint for status to test with the loaders right now, just mock data.